alicloud/provider.go

```diff
@@ -128,6 +128,8 @@ func Provider() terraform.ResourceProvider {
 			"alicloud_eips":                   dataSourceAlicloudEips(),
 			"alicloud_key_pairs":              dataSourceAlicloudKeyPairs(),
 			"alicloud_kms_keys":               dataSourceAlicloudKmsKeys(),
+			"alicloud_kms_ciphertext":         dataSourceAlicloudKmsCiphertext(),
+			"alicloud_kms_secret":             dataSourceAlicloudKmsSecret(),
```

Comment:
I think use alicloud_kms_plaintext will be better.

Reply:
Ok, I renamed `alicloud_kms_secret` to `alicloud_kms_plaintext`